drivers/btrfs/btrfs.go

```diff
@@ -27,8 +27,10 @@ import (
 	"github.com/containers/storage/pkg/idtools"
 	"github.com/containers/storage/pkg/mount"
 	"github.com/containers/storage/pkg/parsers"
+
```

Comment:
At some point, various source files appear to have been grouping import statements into standard library, packages that are part of this repository, and vendored code, but it's inconsistent.  I had started trying to clean that up, but it can wait.